CenterForOpenScience/osf.io
# [Migration Required][QA Prep Required] Feature/Fix incorrect last_logged date on existing forks [PLAT-1131]

❗️ Before merging, make sure this is up-to-date with develop! (is in same release as https://github.com/CenterForOpenScience/osf.io/pull/8808/ - which also has migrations)

## Purpose

Currently, newly created forks receive an incorrect last_logged date. We should find the affected forks and fix the dates.

## Changes

Migration iterates over all forks and find those where the most recent log is the fork log and sets last_logged date to the date of that log

## QA Notes

Create a few forks and note the last_logged date, as well as noting the last logged date of a few nodes.   Make sure the last action that has occurred on the forks was the node_forked action.  Verify that last_logged is the same date as the node_forked action.  If the last log was not related to forks the last_logged date should not have changed.

## Documentation

None needed 

## Side Effects



## Ticket

https://openscience.atlassian.net/browse/PLAT-1131
